buy_together_app/related_to_product/products_views.py

```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render
+from .. import models
+
+
+def index(request):
+    if request.method == 'GET':
+        query_key = 'q'
```

Comment:
what is that?

Reply:
It is a get request for the page 127.0.0.1:8000/products/ and in the page there is search button that makes a query
and the query key is q something like 127.0.0.1:8000/products/q=Apple

Reply:
I don't fully understand, but it sounds like something that should be done in a different way.

Reply:
My feeling as well. Let's discuss it here: https://github.com/redhat-beyond/beyond08-group3/issues/39